front/app/modules/commercial/analytics/admin/typings.ts

```diff
@@ -20,6 +20,39 @@ export interface Pagination {
   pageNumber: number;
 }
 
+// Response values
```

Comment:
I think something went wrong during merging somewhere- all of these types can be deleted, and are failing the dead code CI step

Reply:
Hmm, yeah I didn't add those!